app/views/newsletters/index.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="newsletter-text">
+  <p>
+    Want to be kept up to date with the latest news from Any Rat Rescue? 
+  </p>
+  <button class="subscribe"><%= link_to "Subscribe", "#" %></button>
```

Comment:
I think this is using bitters as well? https://github.com/JessSumner/any_rat_rescue/tree/master/app/assets/stylesheets/base